### PR TITLE
Fix a documentation bug, and add 3 reference links

### PR DESCRIPTION
**Pull request by smorin**
There is a bug referencing the wrong repo in one instance.  Added three references to make it easier for readers of the documentation.

### CODE REVIEW

File: getting-started/mix-otp/introduction-to-mix.markdown [hunk 1]
```diff
@@ -33,17 +33,17 @@ In order to build our key-value application, we are going to use three main tool
...
+* ***[ExUnit](https://hexdocs.pm/ex_unit/)*** is a test-unit based framework that ships with Elixir;
 
 In this chapter, we will create our first project using Mix and explore different features in  <abbr title="Open Telecom Platform">OTP</abbr>, Mix and ExUnit as we go.
 
 > This guide requires Elixir v1.6.1 or later. You can check your Elixir version with `elixir --version` and install a more recent version if required by following the steps described in [the first chapter of the Getting Started guide](/install.html).
 >
 > If you have any questions or improvements to the guide, please reach discussion channels such as the [Elixir Forum](https://elixirforum.com) or the [issues tracker](https://github.com/elixir-lang/elixir-lang.github.com/issues). Your input is really important to help us guarantee the guides are accessible and up to date!
 >
-> The final code for this guide is in [this repository](https://github.com/josevalim/kv_umbrella) and can be used as a reference.
+> The final code for this guide is in [this repository](https://github.com/elixir-lang/elixir-lang.github.com) and can be used as a reference.
```
Comment (michalmuskala) [inline getting-started/mix-otp/introduction-to-mix.markdown]: I think the previous link was correct - it's supposed to be the source code for the thing you build throughout the guide, not the guide itself. 

Maybe we e should make the wording clearer?

**Comment by josevalim** [comment]
Good call. We should say “the final code for the application built on this
guide” or similar. We probably don’t need to link to the guide itself as we
do so at the end of every page.
-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by smorin** [comment]
Updated as requested

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**